src/js/controllers/chatController.js

```diff
@@ -79,14 +79,17 @@ module.exports = ViewController.extend({
         conversationDeferred.then(function(conversation) {
             // update the user before sending the message to ensure properties are correct
             this.user.save({}, {
-                wait: true
+                wait: true,
+                parse: true
```

Comment:
good point, not sure we need it on that one. We have a custom parse function, because the server sometimes returns only the id.